### PR TITLE
migrate:latest error received

### PR DESCRIPTION
**Pull request by CourtneyJa**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [x ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ x] There are no merge conflicts

### CODE REVIEW

File: data/migrations/20200426223901_music.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    tbl.increments();
+    tbl.string("username", 255).unique().notNullable();
+    tbl.string("password", 255).notNullable();
+    tbl.string("email", 255).unique().notNullable();
+    tbl.string("first_name", 255).notNullable();
+    tbl.string("last_name", 255).notNullable();
+  });
+  await knex.schema.createTable("images", (tbl) => {
+    tbl.increments();
+    tbl.text("image_name", 255).notNullable();
```
Comment (jengopockets) [inline data/migrations/20200426223901_music.js:12]: Wondering why text and not string?
```suggestion
    tbl.text("image_name", 255).notNullable();
```

File: data/migrations/20200426223901_music.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    tbl.text("image_category", 255).notNullable();
```
Comment (jengopockets) [inline data/migrations/20200426223901_music.js:13]:
```suggestion
    tbl.string("image_category", 255).notNullable();
```
Wondering why these are text not string?

File: data/migrations/20200426223901_music.js (hunk 1 above)
Comment (jengopockets) [inline data/migrations/20200426223901_music.js:12]:
```suggestion
    tbl.string("image_name", 255).notNullable();
```

File: data/migrations/20200426223901_music.js (hunk 2 above)
Comment (CourtneyJa) [reply data/migrations/20200426223901_music.js:13]: This has been updated for consistency - all applicable changed to string for type

**Pull request merged**